winkelgebieden/winkelgebieden.json

```
@@ -26,20 +26,23 @@
           "geometry": {
             "$ref": "https://geojson.org/schema/Geometry.json"
           },
-          "gebcode": {
-            "type": "string"
+          "gebiedscode": {
+            "type": "string",
+            "provenance": "gebcode"
           },
-          "gebnaam": {
-            "type": "string"
+          "gebiedsnaam": {
+            "type": "string",
+            "provenance": "gebnaam"
           },
           "code": {
             "type": "string"
           },
           "oppha": {
```

Zou dit niet oppervlakte in hectare zijn?

Dan kan je het modelleren als 
```suggestion
          "oppervlakte": { 
            "type": "number",
            "provenance": "oppha",
            "unit": "har"
```

---

Goed gezien, ik ga het aanpassen